### PR TITLE
WT-7091 Restrict lsm tests with compatible backup increment.

### PR DESCRIPTION
**Pull request by raviprakashgiri29**

### CODE REVIEW

**Review by sueloverso — APPROVED** [review]
LGTM

File: test/format/config.c [hunk 1]
```diff
@@ -760,6 +757,23 @@ config_in_memory_reset(void)
     }
 }
 
+/*
+ * config_backup_incr_compatibility_check --
+ *     Backup incremental compatibility check.
+ */
+static void
+config_backup_incr_compatibility_check(void)
```
Comment (kommiharibabu) [inline test/format/config.c:765]: How about specifying the `log` in the function name and description? This function deals with incremental=log.

**Review by kommiharibabu — APPROVED** [review]
LGTM. A small function rename suggestion.

File: test/format/config.c (hunk 1 above)
Comment (raviprakashgiri29) [reply test/format/config.c]: Updated the function name and the function description.

**Pull request merged**